tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -9834,6 +9834,11 @@ declare namespace ts {
         emit(writeFile?: WriteFileCallback, customTransformers?: CustomTransformers): EmitResult | BuildInvalidedProject<T> | undefined;
     }
     type InvalidatedProject<T extends BuilderProgram> = UpdateOutputFileStampsProject | BuildInvalidedProject<T> | UpdateBundleProject<T>;
+    namespace codefix {
+        function _containsJSDocTypedef(node: Node): node is HasJSDoc;
+        function getJSDocTypedefNode(node: HasJSDoc): JSDocTypedefTag;
+        function containsTypeDefTag(node: Node): node is JSDocTypedefTag;
+    }
```

Comment:
This indicates that you are exporting things but not marking them as internal. I would look at other code fixes for examples.

Reply:
@jakebailey Fixed.